bitnami/thanos/Chart.yaml

```diff
@@ -15,4 +15,4 @@ maintainers:
 name: thanos
 sources:
   - https://github.com/bitnami/bitnami-docker-thanos
-version: 0.3.1
\ No newline at end of file
+version: 0.3.2
```

Comment:
Please rebase your branch from master and bump again the version, but this bump the "minor" version since this PR is introducing new parameters. 

Please also add a "newline" at the end of the file as requested in the error above.

Reply:
Done